.travis.yml

```diff
@@ -36,6 +37,13 @@ install:
         && make SYSCFLAGS="-DLUA_USE_LINUX -ULUA_COMPAT_5_2 -DLUA_USE_APICHECK" SYSLIBS="-Wl,-E -ldl -lreadline" LUA_A=liblua.so MYCFLAGS="-fPIC" RANLIB=: AR="gcc -shared -ldl -o" liblua.so \
         && cd .. \
         && sudo make INSTALL_TOP=/usr/ INSTALL_INC=${LUAINCLUDE} TO_LIB=liblua.so linux install)
+    elif [[ "$LUA" == "5.4" ]]; then
+      travis_retry wget https://www.lua.org/work/lua-5.4.0-work2.tar.gz -O lua.tar.gz
+      tar -xvzf lua.tar.gz
+      (cd lua-5.4.0-work2/src \
+        && make SYSCFLAGS="-DLUA_USE_LINUX -ULUA_COMPAT_5_2 -DLUA_USE_APICHECK" SYSLIBS="-Wl,-E -ldl -lreadline" LUA_A=liblua.so MYCFLAGS="-fPIC" RANLIB=: AR="gcc -shared -ldl -o" liblua.so \
```

Comment:
```suggestion
        && make SYSCFLAGS="-DLUA_USE_LINUX -DLUA_COMPAT_5_3 -DLUA_USE_APICHECK" SYSLIBS="-Wl,-E -ldl -lreadline" LUA_A=liblua.so MYCFLAGS="-fPIC" RANLIB=: AR="gcc -shared -ldl -o" liblua.so \
```

Reply:
Why? The idea is to catch uses of deprecated "stuff".

Reply:
-DLUA_COMPAT is always the previous version. So 5.4 it will be 5.3. It is in the lua source code.